src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -888,6 +890,7 @@
                 accessToken = sourceToken,
                 githubPat = targetToken,
                 skipReleases = false,
+                targetRepoVisibility = (string)null,
```

Comment:
The CodeQL warning is legit here, no need to extra cast to `string`.

Reply:
this is necessary because it's an anonymous type and without the cast it doesn't know what type to use for the null (it actually won't compile without the cast IIRC)